mscore/migration/iscoremigrationhandler.h

```diff
@@ -0,0 +1,15 @@
+#ifndef ISCOREMIGRATIONHANDLER_H
+#define ISCOREMIGRATIONHANDLER_H
+
+#include "libmscore/score.h"
+
+class IScoreMigrationHandler
+{
+public:
+    IScoreMigrationHandler() = default;
+    virtual ~IScoreMigrationHandler() = default;
+
+    virtual bool handle(Ms::Score* score) = 0;
```

Comment:
At this moment, the migration handlers have one argument only <code>Ms::Score*</code>. However, the Instrument Ordering handler will required another argument <code>Ms::MuseScore* mscore</code> (required to open the Instrument form which is part of the <code>MuseScore</code> class).

Do you prefer a new PR for this change or shall I add a remark at all files to be modified?

Reply:
Looks like it's only needed for instrument handlers, so I suggest to use Ms::mscore directly, it's an external variable

All you need here is to include "mscore/musescore.h"